carapace-server/src/main/java/org/carapaceproxy/cluster/impl/ZooKeeperGroupMembershipHandler.java

```diff
@@ -187,6 +188,17 @@ public void stop() {
         callbacksExecutor.shutdown();
     }
 
+    @Override
+    public void executeInMutex(String mutexId, Runnable runnable) throws Exception {        
+        InterProcessMutex mutex = new InterProcessMutex(client, "/proxy/mutex/" + mutexId);
+        try {
+            mutex.acquire();
```

Comment:
in case we cannot acquire the Mutex we can simply exit without reporting an error.
This should be stated in higher level interface GroupMembershipHandler.java